README.rst

```diff
@@ -75,9 +75,10 @@ Some of these tests may fail if you:
 Static Code Checkers
 --------------------
 
-You can ensure that code complies to Twisted `coding standards <https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html>`_::
+You can ensure that code complies to Twisted `coding 
+dards <https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html>`_::
```

Comment:
I think that for coding standard we should use "latest" instead of "stable"

Also, we now have a new domain

```suggestion
You can ensure that code complies to Twisted `coding standards <https://docs.twisted.org/en/latest/development/coding-standard.html>`_::
```